CONTRIBUTING.md

```diff
@@ -19,8 +19,9 @@ our own CI/CD: the ["superchain" image][superchain] from.
 The image can be built for local usage, too:
 
 ```console
+$ cd jsii # go to the root of the jsii repo
```

Comment:
```suggestion
```
You can add this to the textual description ("Run from the root of the jsii repo") if you like, but not as a command please.